commands.go

```diff
@@ -171,15 +242,14 @@ func newQemuHelper(c Command) qemuHelper {
 		return q
 	}
 
-	switch c.Architecture {
-	case "armhf", "armel", "arm":
-		q.qemusrc = "/usr/bin/qemu-arm-static"
-	case "arm64":
-		q.qemusrc = "/usr/bin/qemu-aarch64-static"
-	case "amd64", "i386":
-		/* Dummy, no qemu */
-	default:
-		log.Panicf("Don't know qemu for Architecture %s", c.Architecture)
+	qemu, err := GetQemuUser(c.Architecture)
+
+	if err != nil {
+		log.Panicf("%v", err)
```

Comment:
No panic here please!

Reply:
It's what the current code does in the equivalent situation. Sorry, I don't really know Go; is changing the signature of newQemuHelper so it can throw an error a valid thing to do, or would that be an ABI break?